Lesson14/baljan/script.js

```diff
@@ -0,0 +1,214 @@
+const userResponse = prompt("What is your age?").trim();;
+const userResposeNumber = parseInt(userResponse);
+
+if (isNaN(userResposeNumber)) {
+  throw new Error("Please enter a valid number");
+} else {
+  console.log('userResponse',  Math.floor(userResposeNumber));
+}
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+// CODE HERE
+let isStudent = true;
+console.log(isStudent);
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+// CODE HERE
+let counter = 0;
+counter++;
+console.log(counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+// CODE HERE
+let points = 10;
+points += 5;
+console.log(points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+let name = "Alice"
+let age = 30;
+let city = "Paris";
+console.log(`${name} (${age}) lives in ${city}`);
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+let x = 5;
+let y = 10;
+let z = 15;
+let totalSum = x + y + z;
+
+console.log(totalSum);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+let dividend = 10;
+let divisor = 3;
+let divisionResult = dividend / divisor;
+let differenceResult = dividend - divisor;
+
+console.log("Quotient:", divisionResult);
+console.log("Difference:", differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+const firstName = 'Baljan';
+const lastName = 'Kanatbek';
+const fullName = firstName + " " + lastName; ;
+
+console.log(fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+const firstFactor = 7;
+const secondFactor = 2;
+const product = firstFactor * secondFactor;
+
+console.log(product);
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+console.log(Math.PI);
+
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+let count = 0;
+
+count++;
+console.log(count);
+
+count+=1;
+console.log(count);
+
+count=count+1;
+console.log(count);
+
+
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+// CODE HERE
+let initialTemperature = 20;
+initialTemperature += 5;
+console.log(initialTemperature);
+
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+// CODE HERE
+let numberEx9 = 6;
+
+console.log("Before increment: ", numberEx9);
+console.log("After increment: ", ++numberEx9);
+
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+// CODE HERE
+let numberEx10 = 8;
+
+console.log("Before increment: ", numberEx10);
+console.log("After increment: ", ++numberEx10);
```

Comment:
Here, you need to use postfix increment
```suggestion
console.log("After increment: ", numberEx10++);
```